tokio-timer/src/delay_queue.rs

```diff
@@ -161,9 +161,11 @@ pub struct Expired<T> {
     data: T,
 
     /// The expiration time
+    #[allow(unused)]
```

Comment:
Why it's OK in the past?

Reply:
Seems the warnings from other crates will not be displayed when compiling.